src/abi_impl.rs

```diff
@@ -98,3 +102,28 @@ pub(crate) fn assembly_script_print(env: &Env, arg: i32) {
         abi_bail!("Cannot read message pointer in memory");
     }
 }
+
+pub(crate) fn assembly_script_create_sc(env: &Env, bytecode: i32) -> i32 {
+    sub_remaining_point(env, settings::metering_create_sc());
```

Comment:
in a followup, the cost should depend on how long the bytecode is. We don't want someone to fill 1TB for cheap

Reply:
done forward!